classes/blocks/class-socialmedia.php

```diff
@@ -107,40 +108,30 @@ public function __construct() {
 	 *
 	 * @return array The data to be passed in the View.
 	 */
-	public function prepare_data( $fields ): array {
+	private static function get_data( $fields ): array {
 		$title             = $fields['title'] ?? '';
 		$description       = $fields['description'] ?? '';
 		$url               = $fields['social_media_url'] ?? '';
 		$embed_type        = $fields['embed_type'];
 		$alignment_class   = $fields['alignment_class'];
 		$facebook_page_tab = $fields['facebook_page_tab'];
 
-		$data = [
```

Comment:
This was no longer needed since we now use `array_merge` in the [render_callback](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/623/files#diff-749ecefa2f039c9f804d688723f97b5bef853b219013077e09311f6b70395cc4R55)